terra/compute/utils.py

```diff
@@ -92,7 +93,12 @@ def _connect_backend(self):
 For the most part, workflows will be interacting with :data:`compute` to
 ``run`` services. Easier access via ``terra.compute.compute``
 '''
-
+terra.core.signals.logger_configure.connect(
```

Comment:
Do you need to use the lambda trick here since Compute doesn't do the same backend tricks that Executor does?

Reply:
I think since Computer is not a _class_ handler, that is correct